src/ngraph_builder.cc

```diff
@@ -475,9 +503,9 @@ tf::Status Builder::TranslateGraph(const std::vector<tf::TensorShape>& inputs,
         }
         auto concat_op = make_shared<ngraph::op::Concat>(tmp_tensors, 0);
         if (n_dims == 3) {
-          ng_op_map[op->name()] = concat_op;
+          SaveNgOp(ng_op_map, op->name(), concat_op);
         } else {
-          ng_op_map[op->name()] = make_shared<ngraph::op::Reshape>(concat_op, ng::AxisVector{0, 1, 2 }, output_shape);
+          SaveNgOp(ng_op_map, op->name(), make_shared<ngraph::op::Reshape>(concat_op, ng::AxisVector{0, 1, 2 }, output_shape));
```

Comment:
Why aren't you running the clang-formatter? That should take care of fixing lines over 80 characters wide.